MovieDB/Support/MovieCategoryType.swift

```diff
@@ -6,9 +6,70 @@
 //  Copyright © 2019 nguyen.van.hungd. All rights reserved.
 //
 
-enum MovieCategoryType: String {
-    case popular = "popular"
-    case nowPlaying = "now_playing"
-    case upcoming = "upcoming"
-    case topRate = "top_rated"
+//enum MovieCategoryType: String {
+//    case popular = "popular"
+//    case nowPlaying = "now_playing"
+//    case upcoming = "upcoming"
+//    case topRate = "top_rated"
+//
+//    func getCategoryTitle() -> String {
+//        var title = ""
+//        switch self {
+//        case .popular:
+//            title = "Popular"
+//        case .nowPlaying:
+//            title = "Now playing"
+//        case .upcoming:
+//            title = "Upcoming"
+//        case .topRate:
+//            title = "Top rate"
+//        }
+//        return title
+//    }
+//}
+
+enum CategotyType {
+    case popular([Movie])
+    case nowPlaying([Movie])
+    case upcoming([Movie])
+    case topRate([Movie])
+    
+    func getUrlString() -> String {
```

Comment:
```suggestion
    func getUrlString() -> String {
```

chuyển thành property
var urlString: String { get }
tương tự cho getCategoryTitle, getMovies